src/utils/config.ts

```diff
@@ -134,5 +135,6 @@ export function loadConfig() {
       config.length_of_time_for_transport_token_clear ||
       DEFAULT_LENGTH_DELAY_FOR_TRANSPORT_TOKEN_DB_CLEARING,
     boltwall_server: config.boltwall_server || DEFAULT_BOLTWALL_SERVER,
+    private_graph: DEFAULT_PRIVATE_GRAPH,
```

Comment:
so these private graphs should be set per tribe, but it seems like here we are just setting a single value in the config.

If I were to be in two different company tribes then this would not work

Reply:
Yeah!!!.....

This won't work, when I raised that with Paul he said, for now, we are believing a user belongs to only one tribe.
After this, we would focus on other complex stuffs we want to factor in, such as what you just addressed